webapp/src/ts/modals/session-expired/session-expired.component.html

```diff
@@ -0,0 +1,10 @@
+<mm-modal
+  id="'session-expired'"
+  status="status"
```

Comment:
Change this to `[status]="status"` otherwise you're passing the string "status".

Reply:
I just removed the status property, it is not needed at all.